internal/testing/fs/fs.go

```diff
@@ -11,11 +11,9 @@ var _ fs.FS = &FS{}
 // FS emulates fs.FS. Note: the path (map key) cannot begin with "/"!
 type FS map[string]*File
 
-// Open implements the same method as documented on fs.FS.
+// Open implements the same method as documented on fs.FS, except it doesn't
+// validate the path.
 func (f FS) Open(name string) (fs.File, error) {
-	if !fs.ValidPath(name) {
```

Comment:
this is internal and we have no reason to validate, especially as we recently had to add code to make sure we allow fs.FS which don't validate!